### PR TITLE
product name

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Kim noted that each file should use "API Community Manager (ACM)" the first time, then "ACM" every other time. I implemented this here. (Also noticed a small typo and fixed it.)

### CODE REVIEW

**Review by hannanelson — APPROVED** [review]
LGTM!

File: modules/ROOT/pages/install-acm.adoc [hunk 1]
```diff
@@ -4,9 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-NOTE: Installing, updating or configuring API Community Manager must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
+NOTE: Installing, updating or configuring API Community Manager (ACM) must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-acm.adoc:7]: Intentionally passive?
Also, consider just merging this requirement into the task intro below.

File: modules/ROOT/pages/install-acm.adoc [hunk 2]
```diff
@@ -4,9 +4,9 @@ include::_attributes.adoc[]
...
 
-To perform a new install of the API Community Manager package in your ACM Salesforce Organization, follow the steps below. To update an existing API Community Manager installation, refer to xref:update-acm.adoc[Update ACM].
+To perform a new install of the API Community Manager package in your ACM Salesforce Organization, follow the steps below. To update an existing ACM installation, refer to xref:update-acm.adoc[Update ACM].
```
Comment (kbostrom) [inline modules/ROOT/pages/install-acm.adoc:9]: install operation or installation, not install

Comment (kbostrom) [reply modules/ROOT/pages/install-acm.adoc:9]: Note that organization is not capitalized in previous text but is here; GLOBALLY standardize. I suspect this should be lowercase.

Comment (kbostrom) [reply modules/ROOT/pages/install-acm.adoc:9]: GLOBALLY reword to not refer to anything "above" or "below." Use following, next, previous, earlier, and so on.

File: modules/ROOT/pages/install-acm.adoc (hunk 1 above)
Comment (kbostrom) [reply modules/ROOT/pages/install-acm.adoc:7]: Also, use the Oxford comma: Installing, updating, or configuring

File: modules/ROOT/pages/install-configure.adoc [hunk 3]
```diff
@@ -4,9 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-NOTE: Installing, updating or configuring API Community Manager must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
+NOTE: Installing, updating or configuring API Community Manager (ACM) must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-configure.adoc:7]: ditto previous edits to this same note earlier

File: modules/ROOT/pages/licensing-overview.adoc [hunk 4]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) licenses are procured and provisoned by MuleSoft. A standard ACM License consists of a *Salesforce Customer Community Plus Login License Unlimited Edition*. It is based on the number of member logins per month, and includes as a base:
+API Community Manager (ACM) licenses are procured and provisioned by MuleSoft. A standard ACM License consists of a *Salesforce Customer Community Plus Login License Unlimited Edition*. It is based on the number of member logins per month, and includes as a base:
```
Comment (kbostrom) [inline modules/ROOT/pages/licensing-overview.adoc:7]: Intentionally passive?

Comment (kbostrom) [reply modules/ROOT/pages/licensing-overview.adoc:7]: lowercase "license"

Comment (kbostrom) [reply modules/ROOT/pages/licensing-overview.adoc:7]: To what does "It" refer: standard license or edition?

**Review by kbostrom — COMMENTED** [review]
Reword to active voice and try to avoid using Notes for text that can be incorporated into the running narrative with ease.

**Pull request merged**